templates/partials/tab-background.html

```diff
@@ -16,10 +16,7 @@
                 value="{{details.value}}" />
             {{/each}}
         </div>
-        
-        <div class="lifeEvents background">
-            <textarea rows=25 name="data.general.background.value">{{data.general.background.value}}</textarea>
-        </div>    
+        {{editor content=data.general.background.value target="data.general.background.value" button=true editable=editable}}
```

Comment:
Good idea, I wonder if it would be better to have this editor in other sheets like for notes, and monster Knowledge

Reply:
Yeah worth trying :)